util/cc2file.go.tmpl

```diff
@@ -0,0 +1,34 @@
+{{/*
+	This command sends your CC(s) code in a text file, rather than "plain" Discord messages, preserving Tabs, markdown, etc. 
+	You can especify more than one ID or trigger, and bot will generate one file per input parameter. If none are provided, output will be CC list
+	
+	Usage: `-cc2file <ID or Trigger> [ID or Trigger] [ID or Trigger]...`. For example: `-cc2file 10 "te st" 23`
+	Recommended trigger: Command trigger with trigger `cc2file`.
+	Configuration variables: None
+*/}}
+
+
+{{range .CmdArgs}}
+	{{- $exec_cc_Answer:= exec "cc" .}}
```

Comment:
You seem to be using camelCase for variables except this one, could you rename it for consistency?

Reply:
Yup! In updated code, I've directly removed that variable. I was reading its content just one time